frontend_lib/src/component/RefreshWarningMessage/RefreshWarningMessage.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'react-i18next'
+
+const RefreshWarningMessage = props => (
+  <div className='refreshWarningMessage'>
+    <i className='fa fa-exclamation-triangle' />
+    {props.t('The content has been modified by {{author}}', { author: props.editionAuthor, interpolation: { escapeValue: false } })}
+    <button
+      className='refreshWarningMessage__button'
+      onClick={props.onClickRefresh}
+      title={props.t('If you refresh, you will lose the current changes')}
+    >
+      {props.t('Refresh?')}
+    </button>
+  </div>
+)
+
+export default translate()(RefreshWarningMessage)
+
+RefreshWarningMessage.propTypes = {
+  editionAuthor: PropTypes.string,
+  onClickRefresh: PropTypes.func
+}
+
+RefreshWarningMessage.defaultState = {
```

Comment:
It's not `defaultState`, it's `defaultProps`

Reply:
I just looked that up on our code base and I found 3 instances of that mistake:
- frontend_lib/src/component/FileDropzone/FileUploadList.jsx
- frontend_lib/src/component/DisplayState/DisplayState.jsx
- frontend_lib/src/component/FileDropzone/FileDropzone.jsx

I'd like you to changes these too please

Reply:
Changed